.eslintrc.yml

```diff
@@ -79,8 +81,8 @@ rules:
   prefer-spread: error
   quote-props: [error, as-needed]
   quotes: [error, single, avoid-escape]
-  semi: [error, always]
-  semi-spacing: error
+  # semi: [error, never]
+  # semi-spacing: error
```

Comment:
why commented?

Reply:
@DedaDev To temporarily suspense linter error till I remove all semicolons.